File: web/web service/DAO/Cliente.php
<?php 

class Cliente {

	//Atributos
	private $idCliente;
	private $nombre;
	private $correo;
	private $telefono;
	private $genero;
	private $limiteSaldo;
	private $saldoDisponible;

	//Constructores
	function __construct() {

	}

	function Cliente($nombre, $correo, $telefono, $genero, $limiteSaldo, $saldoDisponible) {
		this::$nombre = $nombre;
		this::$correo = $correo;
		this::$telefono = $telefono;
		this::$genero = $genero;
		this::$limiteSaldo = $limiteSaldo;
		this::$saldoDisponible = $saldoDisponible;
	}

	//Métodos constructores
	public static function getIdCliente() {
		return this::$idCliente;
	}

	public static function setIdCliente($idCliente) {
		this::$idCliente = $idCliente;
	}

	public static function getNombre() {
		return this::$nombre;
	}

	public static function setNombre($nombre) {
		this::$nombre = $nombre;
	}

	public static function getCorreo() {
		return this::$correo;
	}

	public static function setCorreo($correo) {
		this::$correo = $correo;
	}

	public static function getTelefono() {
		return this::$telefono;
	}

	public static function setTelefono($telefono) {
		this::$telefono = $telefono;
	}

	public static function getGenero() {
		return this::$genero;
	}

	public static function setGenero($genero) {
		this::$genero = $genero;
	}

	public static function getLimiteSaldo() {
		return this::$limiteSaldo;
	}

	public static function setLimiteSaldo($limiteSaldo) {
		this::$limiteSaldo = $limiteSaldo;
	}

	public static function getSaldoDisponible() {
		return this::$saldoDisponible;
	}

	public static function setSaldoDisponible($saldoDisponible) {
		this::$saldoDisponible = $saldoDisponible;
	}

}

?>

File: core/Core/src/modelo/DAO/Cliente.java
package modelo.DAO;

/**
 *
 * @author DDD
 */

public class Cliente {

    //<editor-fold defaultstate="collapsed" desc="Atributos">
    private int idCliente;
    private String nombre;
    private String correo;
    private String telefono;
    private String genero;
    private String noTarjeta;
    private double limiteSaldo;
    private double saldoDisponible;
    private Usuario usuario;
    //</editor-fold>

    //<editor-fold defaultstate="collapsed" desc="Constructores">
    /**
     * Constructor por defecto
     */
    public Cliente() {
    }

    /**
     * Constructor sobrecargado
     * 
     * @param nombre
     * @param correo
     * @param telefono
     * @param genero
     * @param nTarjeta
     * @param limiteSaldo
     * @param saldoDisponible
     * @param idUsuario
     */
    public Cliente(String nombre, String correo, String telefono, String genero, String nTarjeta, double limiteSaldo, double saldoDisponible, Usuario idUsuario) {
        this.nombre = nombre;
        this.correo = correo;
        this.telefono = telefono;
        this.genero = genero;
        this.noTarjeta = nTarjeta;
        this.saldoDisponible = saldoDisponible;
        this.limiteSaldo = limiteSaldo;
        this.usuario = idUsuario;
    }
    //</editor-fold>

    //<editor-fold defaultstate="collapsed" desc="Métodos get y set">
    /**
     * Método get del usuario
     * 
     * @return Usuario
     */
    public Usuario getUsuario() {
        return usuario;
    }

    /**
     * Método set del usuario
     * 
     * @param usuario
     */
    public void setUsuario(Usuario usuario) {
        this.usuario = usuario;
    }

    /**
     * Método get id de usuario
     * 
     * @return Id del usuario
     */
    public int getIdCliente() {
        return idCliente;
    }

    /**
     * Método set del id de usuario
     * 
     * @param idCliente
     */
    public void setIdCliente(int idCliente) {
        this.idCliente = idCliente;
    }

    /**
     * Método get del nombre
     * 
     * @return Nombre
     */
    public String getNombre() {
        return nombre;
    }

    /**
     * Método set del nombre
     * 
     * @param nombre
     */
    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    /**
     * Método get del correo
     * 
     * @return Correo
     */
    public String getCorreo() {
        return correo;
    }

    /**
     * Método set del correo
     * 
     * @param correo
     */
    public void setCorreo(String correo) {
        this.correo = correo;
    }

    /**
     * Método get del teléfono
     * 
     * @return Teléfono
     */
    public String getTelefono() {
        return telefono;
    }

    /**
     * Método set del teléfono
     * 
     * @param telefono
     */
    public void setTelefono(String telefono) {
        this.telefono = telefono;
    }

    /**
     * Método get genero
     * 
     * @return Genero
     */
    public String getGenero() {
        return genero;
    }

    /**
     * Método set del genero
     * 
     * @param genero
     */
    public void setGenero(String genero) {
        this.genero = genero;
    }

    /**
     * Método get número de tarjeta
     * 
     * @return Número de tarjeta
     */
    public String getNoTarjeta() {
        return noTarjeta;
    }

    /**
     * Método set del número de tarjeta
     * 
     * @param noTarjeta
     */
    public void setNoTarjeta(String noTarjeta) {
        this.noTarjeta = noTarjeta;
    }

    /**
     * 
     * 
     * @return 
     */
    public double getLimiteSaldo() {
        return limiteSaldo;
    }

    /**
     * 
     * 
     * @param limiteSaldo 
     */
    public void setLimiteSaldo(double limiteSaldo) {
        this.limiteSaldo = limiteSaldo;
    }

    /**
     * 
     * 
     * @return 
     */
    public double getSaldoDisponible() {
        return saldoDisponible;
    }

    /**
     * 
     * 
     * @param saldoDisponible 
     */
    public void setSaldoDisponible(double saldoDisponible) {
        this.saldoDisponible = saldoDisponible;
    }
    
}